server/schemas/resolvers.js

```diff
@@ -1,4 +1,238 @@
 const { AuthenticationError } = require('apollo-server-express');
-//const { User } = require('../models');
+const { User } = require('../models');
 const { signToken } = require('../utils/auth');
-const stripe = require('stripe')('sk_test_4eC39HqLyjWDarjtT1zdp7dc');
\ No newline at end of file
+const stripe = require('stripe')('sk_test_4eC39HqLyjWDarjtT1zdp7dc');
+
+
+const resolvers = {
+    Query: {
```

Comment:
do we need to find all the users? it might be practical for us but let's leave it here for now